packages/react-components/src/components/chart/chartOptions/seriesAndYAxis/convertSeriesAndYAxis.ts

```diff
@@ -68,9 +68,10 @@ const convertSeries = (
     lineThickness,
     emphasis,
     significantDigits,
+    hidden,
   }: ChartStyleSettingsWithDefaults
 ) => {
-  const opacity = emphasis === 'de-emphasize' ? DEEMPHASIZE_OPACITY : 1;
+  const opacity = hidden ? 0 : emphasis === 'de-emphasize' ? DEEMPHASIZE_OPACITY : 1;
```

Comment:
could we pull this out into a lil function without the nested ternary operator for legibility :)